src/makeCalculator.js

```diff
@@ -4,7 +4,27 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calc = {
+    result: 0,
+    add: (n) => (calc.result += n),
+    subtract: (n) => (calc.result -= n),
+    multiply: (n) => (calc.result *= n),
+    divide: (n) => (calc.result /= n),
+
+    operate: function (callback, number) {
+      callback(number);
+
+      return this;
+    },
+
+    reset: function () {
+      calc.result = 0;
+
+      return this;
```

Comment:
[CODE KNOWLEDGE] - similarly, the `reset` method should use the shorthand method syntax instead of the `function` keyword.

BAD EXAMPLE:
 reset: function () {
  // your code
 },

GOOD EXAMPLE:
 reset() {
  // your code
 },